### ISSUE TITLE
[Accessibility] Insufficient color contrast in discount text on block checkout

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#61762]
### Prerequisites

- [x] I have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

The discounted amount text in the Order Summary on the block checkout is `#4ab866`, which has a contrast ratio of 2.5:1 with a white background.

<img width="1822" height="770" alt="Image" src="https://github.com/user-attachments/assets/67168c1e-eb4a-424e-8965-1fb25c992568" />

[See contrast report](https://webaim.org/resources/contrastchecker/?fcolor=4AB866&bcolor=FFFFFF)

### Expected behavior

The text will have a contrast ratio of at least 4.5:1.

### Actual behavior

Current styles:
```
.wc-block-components-totals-discount .wc-block-components-totals-item__value {
    color: #4ab866;
}
```

### Steps to reproduce

1. Set up a site with a block checkout.
2. Create a coupon code.
3. Add a product to the cart.
4. Go to the checkout page.
5. Add the coupon code in the coupon field.
6. Inspect the discount amount to see the hex code.

### WordPress Environment

<details><summary>System Status Report</summary>

```
### WordPress Environment ###

WordPress address (URL): [Redacted]
Site address (URL): [Redacted]
WC Version: 10.3.4
Legacy REST API Package Version: The Legacy REST API plugin is not installed on this site.
Action Scheduler Version: ✔ 3.9.3
Log Directory Writable: ✔
WP Version: 6.8.3
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache
Server Architecture: Linux 4.19.286-203.ELK.el7.x86_64 x86_64
PHP Version: 8.1.28
PHP Post Max Size: 516 MB
PHP Time Limit: 60
PHP Max Input Vars: 1000
cURL Version: 8.7.1
OpenSSL/1.1.1w

SUHOSIN Installed: –
MySQL Version: 5.7.23-23
Max Upload Size: 512 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

[REDACTED]

### Post Type Counts ###

attachment: 64
aw_workflow: 5
page: 17
post: 2
product: 68
product_variation: 163
revision: 56
shop_coupon: 5
shop_order_placehold: 136
wp_global_styles: 2
wp_navigation: 1
wp_template: 2

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (27) ###

AutomateWoo - Birthdays Add-on: by WooCommerce – 1.3.51
AutomateWoo - Refer A Friend Add-on: by WooCommerce – 2.8.9
AutomateWoo: by WooCommerce – 6.1.16
The Bluehost Plugin: by Bluehost – 4.7.1
Code Snippets: by Code Snippets Pro – 3.8.1
Password Protected: by Password Protected – 2.7.12
Pinterest for WooCommerce: by WooCommerce – 1.4.22
PU2 DevTools: by  –
Stream: by XWP – 4.1.1
User Switching: by John Blackbourn – 1.11.0
WooCommerce All Products for Subscriptions: by Woo – 6.0.6
WooCommerce Back In Stock Notifications: by Woo – 3.3.101-dev
WooCommerce Composite Products: by Woo – 10.7.020-dev
WooCommerce Conditional Shipping and Payments: by Woo – 2.2.0
WooCommerce Deposits: by WooCommerce – 2.4.0
WooCommerce EU VAT Number: by WooCommerce – 3.0.2
WooCommerce Gift Cards: by Woo – 2.7.100-dev
WooCommerce Min/Max Quantities: by Woo – 5.2.5
WooPayments: by WooCommerce – 10.1.1
WooCommerce Pre-Orders: by WooCommerce – 2.3.1
WooCommerce Product Add-Ons: by WooCommerce – 8.0.101-dev
WooCommerce Product Bundles: by Woo – 8.5.01-dev
WooCommerce Subscription Downloads: by WooCommerce – 1.4.1
WooCommerce Subscriptions Gifting: by WooCommerce – 2.9.0
WooCommerce Subscriptions: by WooCommerce – 7.6.0
WooCommerce: by Automattic – 10.3.4
WP Mail Logging: by WP Mail Logging Team – 1.15.0

### Inactive Plugins (9) ###

Akismet Anti-spam: Spam Protection: by Automattic - Anti-spam Team – 5.5
Braintree for WooCommerce Payment Gateway: by WooCommerce – 3.4.2
Disable Emails: by WebAware – 1.8.2
Jetpack: by Automattic – 15.1.1
UpdraftPlus - Backup/Restore: by TeamUpdraft
DavidAnderson – 1.25.8

WooCommerce Box Office: by WooCommerce – 1.3.7
WooCommerce Smooth Generator: by Automattic – 1.2.0
WooCommerce Square: by WooCommerce – 5.1.0
WooCommerce Stripe Gateway: by Stripe – 10.0.1

### Must Use Plugins (1) ###

SSO: by Garth Mortensen
Mike Hansen – 0.5


### Settings ###

Legacy API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: bundle (bundle)
composite (composite)
external (external)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔
Enforce Approved Product Download Directories: ✔
HPOS feature enabled: ✔
Order datastore: Automattic\WooCommerce\Internal\DataStores\Orders\OrdersTableDataStore
HPOS data sync enabled: –
Enabled Features: analytics
marketplace
order_attribution
site_visibility_badge
remote_logging
email_improvements
blueprint
point_of_sale
custom_order_tables


### Logging ###

Enabled: ✔
Handler: Automattic\WooCommerce\Internal\Admin\Logging\LogHandlerFileV2
Retention period: 30 days
Level threshold: –
Log directory size: 125 KB

### WC Pages ###

Shop base: #6 - /shop/
Cart: #7 - /cart/ -  Contains the woocommerce/cart block
Checkout: #8 - /checkout/ -  Contains the woocommerce/checkout block
My account: #9 - /my-account/ -  Contains the [woocommerce_my_account] shortcode
Terms and conditions: ❌ Page not set

### Theme ###

Name: Twenty Twenty-Five
Version: 1.3
Author URL: https://wordpress.org
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
Theme type: Block theme
WooCommerce Support: ❌ Not declared

### Templates ###

Overrides: /home2/amberhin/public_html/website_f5ae36ad/wp-content/plugins/woocommerce/templates/block-notices/error.php
/home2/amberhin/public_html/website_f5ae36ad/wp-content/plugins/woocommerce/templates/block-notices/notice.php version 9.5.0 is out of date. The core version is 10.2.0
/home2/amberhin/public_html/website_f5ae36ad/wp-content/plugins/woocommerce/templates/block-notices/success.php

Outdated Templates: ❌
					
					
						Learn how to update					 |
					
						
					
					
						Clear system status theme info cache

### WooPayments ###

Version: 10.1.1
Connected to WPCOM: Yes
WPCOM Blog ID: 246243230
Account ID: acct_1SPA8wCKsYZx3eFY
Payment Gateway: Enabled
Test Mode: Enabled
Enabled APMs: card,affirm,afterpay_clearpay,klarna
WooPay: Enabled (product,cart,checkout)
WooPay Incompatible Extensions: Yes
Apple Pay / Google Pay: Enabled (product,cart,checkout)
Fraud Protection Level: basic
Multi-currency: Enabled
Auth and Capture: Disabled
Documents: Disabled
Logging: Disabled

### Subscriptions ###

WCS_DEBUG: ✔ No
Subscriptions Mode: ✔ Live
Subscriptions Live URL: https://woo.equalizedigital.com
Subscriptions-core Library Version: 8.3.0
Subscription Statuses: wc-active: 1
wc-cancelled: 8
wc-on-hold: 6

WooCommerce Account Connected: ✔ Yes
Active Product Key: ❌ No

### Store Setup ###

Country / State: United States (US) — Texas

### Subscriptions by Payment Gateway ###

Direct bank transfer: wc-cancelled: 1
wc-on-hold: 4

other: wc-on-hold: 1

### Payment Gateway Support ###

WooPayments: products
refunds
multiple_subscriptions
subscription_cancellation
subscription_payment_method_change_admin
subscription_payment_method_change_customer
subscription_payment_method_change
subscription_reactivation
subscription_suspension
subscriptions
subscription_amount_changes
subscription_date_changes
tokenization
add_payment_method

WooPayments: products
refunds
multiple_subscriptions
subscription_cancellation
subscription_payment_method_change_admin
subscription_payment_method_change_customer
subscription_payment_method_change
subscription_reactivation
subscription_suspension
subscriptions
subscription_amount_changes
subscription_date_changes
tokenization
add_payment_method

WooPayments: products
refunds
multiple_subscriptions
subscription_cancellation
subscription_payment_method_change_admin
subscription_payment_method_change_customer
subscription_payment_method_change
subscription_reactivation
subscription_suspension
subscriptions
subscription_amount_changes
subscription_date_changes
tokenization
add_payment_method

WooPayments: products
refunds
multiple_subscriptions
subscription_cancellation
subscription_payment_method_change_admin
subscription_payment_method_change_customer
subscription_payment_method_change
subscription_reactivation
subscription_suspension
subscriptions
subscription_amount_changes
subscription_date_changes
tokenization
add_payment_method

Direct bank transfer: products
Check payments: products
Cash on delivery: products

### Back In Stock ###

Database Version: 3.3.101
Loopback Test: ✔

### Composite Products ###

Database Version: ❌ 10.7.102 - Database version mismatch.
Loopback Test: ✔
Template Overrides: –

### Gift Cards ###

Database Version: 2.7.100
Loopback Test: ✔
Task Queueing Test: ✔

### Product Bundles ###

Database Version: 8.5.01
Loopback Test: ✔
Template Overrides: –

### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
core-profiler
customize-store
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
onboarding
onboarding-tasks
pattern-toolkit-full-composability
product-custom-fields
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
printful
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page
launch-your-store

Disabled Features: product-data-views
experimental-blocks
experimental-iapi-mini-cart
experimental-iapi-runtime
coming-soon-newsletter-template
minified-js
product-pre-publish-modal
settings
async-product-editor-category-field
product-editor-template-system
use-wp-horizon
rest-api-v4
reactify-classic-payments-settings

Daily Cron: ✔ Next scheduled: 2025-11-03 19:10:53 -06:00
Options: ✔
Notes: 126
Onboarding: completed

### Subscriptions Gifting ###

Gifted Subscriptions Count: 4

### All Products for WooCommerce Subscriptions ###

Template Overrides: –

### Action Scheduler ###

Canceled: 4
Oldest: 2025-10-30 11:44:09 -0500
Newest: 2025-11-02 17:23:00 -0600

Complete: 2,903
Oldest: 2025-10-28 22:16:32 -0500
Newest: 2025-11-02 21:09:56 -0600

Failed: 1,427
Oldest: 2024-09-06 00:28:33 -0500
Newest: 2025-11-02 20:27:02 -0600

Pending: 25
Oldest: 2025-11-02 21:10:22 -0600
Newest: 2025-12-08 13:49:23 -0600


### Status report information ###

Generated at: 2025-11-02 21:10:26 -06:00
```
</details>

### Isolating the problem

- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [x] I can reproduce this bug consistently using the steps above.
